src/main/java/org/kainos/ea/DropwizardTheITCrowdServiceApplication.java

```diff
@@ -19,17 +23,26 @@ public String getName() {
 
   @Override
   public void initialize(final Bootstrap<DropwizardTheITCrowdServiceConfiguration> bootstrap) {
+
+    try { // To Speed Up Processing We Generate RSA Keys On Startup.
+      KeyGeneratorUtil.setInstance(new KeyGeneratorUtil());
+    } catch (CouldNotGenerateKeyPairException e) {
+      throw new RuntimeException(e);
```

Comment:
I'm not a fan of throwing a `RuntimeException` here

Reply:
Going to print stacktrace instead. Not sure what would be best. Give as much detail as possible?